### ISSUE TITLE
It's not obvious when you click "Add to Favs" that it's actually added

### ISSUE THREAD

**Issue by remlap42** [broadsoftxtended/Product-Phone-for-Chrome#34]
the button doesn't go away....so the user isn't sure if the action took place.

Could that button simply be a Star?  I guess then you would have to have a state for it by cross referencing the list with the favs...

**Comment by joncodo** [comment]
Ah. You are right. Not sure how to really handle that. Just close the box?

**Comment by remlap42** [comment]
Yeah...if the box went away, ti would be good enough.  What do yo think about the "star" idea?

**Comment by joncodo** [comment]
Its better. And doable

About 1.5 hours I think

**Comment by remlap42** [comment]
I would think that would be a better experience...can get a star icon for cheap, I'm sure.

**Comment by joncodo** [comment]
There is one here: Font-awesome. http://fortawesome.github.io/Font-Awesome/icons/

**Comment by remlap42** [comment]
If you did that, I guess you would have to have the ability to Unstar anywhere the contact shows up....unstarring in the Favs list would make it go away from the favs list

**Comment by joncodo** [comment]
Yeah. I included that in my 1.5 hours. Its a pretty standard feature. Ill tackle that next. The add to favs popup is ugly.

**Comment by remlap42** [comment]
That way when we add google stored favs, it will be the same interface
